pkg/sqlmanager/sqlbuilder.go

```diff
@@ -0,0 +1,54 @@
+package sqlmanager
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/lib/pq"
+	"github.com/nrc-no/core/pkg/sql/schema"
+)
+
+type SQLBuilder interface {
+	InsertRow(
+		schemaName string,
+		tableName string,
+		columns []string,
+		values []interface{},
+	) schema.DDL
+}
+
+type sqlBuilder struct {
+}
+
+func NewSQLBuilder() SQLBuilder {
+	return sqlBuilder{}
+}
+
+func (s sqlBuilder) InsertRow(
+	schemaName string,
+	tableName string,
+	columns []string,
+	values []interface{},
+) schema.DDL {
+	var placeholders = make([]string, len(columns))
```

Comment:
"placeholders" as in "defaults"

Reply:
This is a copy+paste of another function that was unfortunately made private

Reply:
so, are they meant to be defaults or not?

Reply:
I think what it does is make `$1, $2, $3, etc` which correspond to `Args:  values,`. So when the statement is executed `values[0]` is `$1`

Reply:
So not defaults